src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -729,6 +754,13 @@ public void displaySessionAttributes(HttpServletRequest request) {
 		}
 	}
 
+	public void displayRequestHeaders(HttpServletRequest request) {
+		Enumeration headerNames = request.getHeaderNames();
```

Comment:
Check if debug is enabled as the first line of method so don't do anything if not debugging.  Also makes it clear that this method is only for debugging.